rfcs/0050-roadmaps.md

```diff
@@ -0,0 +1,320 @@
+    - Themes are intended to group items within a roadmap. This helps us to understand the scope of
+    the roadmap.
+- For each theme, include a set of definitions specific to the proposed roadmap.
+    - What does this theme mean in the context of this roadmap?
+    - Are there multiple definitions for this theme? For example, `performance` could be interpreted
+    as tuning times, runtime latency, or a number of other definitions.
+- For each theme, include a set of success criteria specific to the proposed roadmap.
+    - What types of metrics would be relevant to this roadmap?
+
+### Adding Items to a Roadmap
+
+A roadmap's maintainers are the primary folks responsible for adding, modifying, and removing items
+for a roadmap. This isn't a hard and fast rule—for example, it may be expedient for other community
+members to triage new roadmap items into a roadmap's backlog. However, the maintainers should be
+considered the "owners" of a roadmap, and generally no rigid process is defined around modifying the
+items in a roadmap.
+
+Roadmaps are defined using [GitHub
+Projects](https://docs.github.com/en/issues/trying-out-the-new-projects-experience/about-projects)
+and can include GitHub Issues, Pull Requests, and simple note cards. Maintainers should strive to
+place mainly **GitHub Issues** in Roadmaps to make it possible for the community to learn more about
+ongoing work and reduce triage burden.
+
+Each item on a roadmap is intended to track one of these community processes:
+
+- **pre-RFCs** serve as a way to begin discussions on planned work at the earliest stage of
+  maturity. **pre-RFCs** are typically posted in the [TVM Discuss
+  forums](http://discuss.tvm.apache.org) in order to solicit TVM Community feedback. For an example
+  of a **pre-RFC**, see the screenshot of Andrew R's proposal to *Convert RST Docs to Markdown*
+  below.
+
+    **pre-RFCs** can be tracked on a Roadmap by preemptively creating a GitHub Task-tracking Issue
+    in [tvm-rfcs](https://github.com/apache/tvm-rfcs).
+    ![image|690x460](./assets/0050/roadmap-item-pre-rfc.png)
+
+- **RFCs** serve as a way to share feature proposals with the TVM Community. The
+  [tvm-rfcs](https://github.com/apache/tvm-rfcs) repo is used for the creation and acceptance of
+  **RFCs**. For some examples of **RFCs**, see the screenshot of open pull requests in
+  [tvm-rfcs](https://github.com/apache/tvm-rfcs) below.
+
+    Open **RFCs** can be directly linked into any roadmap. Once an **RFC** is accepted, please use
+    the **Github Task-Tracking** process to track **RFC** Execution.
+    ![image|690x357](./assets/0050/roadmap-item-rfc.png)
+
+- **Github Task-Tracking Issues** are used in [tvm](https://github.com/apache/tvm) to share the
+  progress of accepted RFCs over time. For an example of a **Github Task-Tracking Issue**, see the
+  screenshot of Andrew L's RFC to *Add Mixed-Precision Support to TVM* below.
```

Comment:
Would be good use the GitHub username here to illustrate which Andrew L further :smile_cat:

Reply:
:%s/Andrew L's/@AndrewZhaoLuo's/g

Reply:
done!